uui/components/navigation/MainMenu/MainMenuDropdown.tsx

```diff
@@ -37,6 +37,8 @@ export function MainMenuDropdown(props: MainMenuDropdownProps) {
                 </FocusLock>
             ) }
             placement="bottom-start"
+            // In the case, the default "dialog" role is not applicable.
+            role={ undefined }
```

Comment:
Should we accept the role from props in this component instead of hard-coding the value, since the body can potentially be anything?